packages/nx/src/utils/nx-plugin.ts

```diff
@@ -23,6 +23,7 @@ import {
 } from '../project-graph/utils/find-project-for-path';
 import { normalizePath } from './path';
 import { join } from 'path';
+import { ensureTypescript } from '@nrwl/js/src/utils/typescript/ensure-typescript';
```

Comment:
This probably needs to actually be used, or removed more likely. nx shouldn't import from @nrwl/js right?